pkg/retry/retry.go

```diff
@@ -9,6 +9,10 @@ const (
 	Interval = 15 * time.Second
 	// Timeout is the total time we will wait for an event to occur.
 	Timeout = time.Minute * 10
+	// TCPConnTimeout is the maximum amount of time for a single TCP connection to establish
+	TCPConnTimeout = time.Second * 45
+	// SSHRetryTimeout is the total time we will retry to establish an SSH connection with a particular socket address
+	SSHRetryTimeout = time.Minute * 5
```

Comment:
These constants seem specific to the connectivity package. I suggest moving them there.